sirepo/package_data/static/json/activait-schema.json

```diff
@@ -291,7 +291,9 @@
             "featureRangeMax": ["Feature range maximum", "Integer", 1],
             "inputsScaler": ["Inputs Scaler", "Scaler", "RobustScaler"],
             "outputsScaler": ["Outputs Scaler", "Scaler", "RobustScaler"],
-            "url": ["Data URL", "URL", ""]
+            "url": ["Data URL", "URL", ""],
+            "exampleFile": ["", "String", ""],
```

Comment:
I don't really like this structure. It doesn't allow for growth in the future. What happens when there are 3 files? I'd get Paul's thoughts on this. But, one idea is we have a name like "exampleDirectory" there is then a name with that directory in sirepo-data-activait. That dir can have an arbitrary number of files. If there is one file then we just save it. If there is more than one file then we combine them. If ordering matters then give the files names like 00.h5, 01.h5, etc and use sorted glob to order and iterate over them.